### PR TITLE
Fix sector GET test

### PR DESCRIPTION
**Pull request by thobson88**
As discussed in nats repo #162

### CODE REVIEW

**Comment by rkm** [comment]
The build will fail due to some other issues, but I'll merge this if the sector tests now pass 👍

**Comment by rkm** [comment]
No luck it seems: https://travis-ci.com/alan-turing-institute/bluebird/builds/144314709#L1283

**Comment by thobson88** [comment]
@rkm For this to work, I need travis to pick up aviary from the `develop` branch, so I tried updating the relevant line in the [travis config](https://github.com/alan-turing-institute/bluebird/pull/83/files) but I can't see this change in the latest [travis build](https://travis-ci.com/alan-turing-institute/bluebird/jobs/275347851/config) - i.e. it still points to the `feature/deserialise-sector` branch.

**Comment by rkm** [comment]
I've cleared the Travis cache and re-added the explicit pip install for Aviary - this time with `--no-cache-dir --force-reinstall`. Hopefully this should fix things.

**Comment by rkm** [comment]
Looks like this is still failing (during test collection now): https://travis-ci.org/alan-turing-institute/bluebird/builds/636916893#L660

**Comment by rkm** [comment]
Sorted! https://travis-ci.org/alan-turing-institute/bluebird/builds/637666096#L605

**Comment by thobson88** [comment]
Nice one @rkm. This is great.

File: tests/unit/sim_proxy/proxy_simulator_controls_test.py [hunk 1]
```diff
@@ -37,7 +38,7 @@
 with open(TEST_SECTOR, "r") as f:
     geojson = json.load(f)
     del geojson["_source"]
-    sector_element = SectorElement.deserialise(json.dumps(geojson))
+    sector_element = SectorElement.deserialise(StringIO(json.dumps(geojson)))
```
Comment (thobson88) [inline tests/unit/sim_proxy/proxy_simulator_controls_test.py]: @rkm Sorry - coming to this late, but do we need to dump and read from a string here? 

It should be possible to pass the `geojson` from line 39 directly to `SectorElement.deserialise` (as in `sector_test.py` above). If not I need to improve the `deserialise` method.

I think this applies to `sector_validation.py` (above) too.

Comment (rkm) [reply tests/unit/sim_proxy/proxy_simulator_controls_test.py]: You're right, but I added a "_source" tag to the test files which references the git hash in Aviary where they were copied from. I *thought* this was sensible to do at the time, although not so sure anymore!

**Pull request merged**